docs/source/aws/gettingstarted.rst

```diff
@@ -49,7 +49,7 @@ Custodian tag filter by simply specifying the ``stop`` action:
         filters:
           - "tag:Custodian": present
         actions:
-          - stop .. _aws-run-policy:
```

Comment:
Thanks, looks like this was incorrectly formatted, the `.. _ aws-run-policy:` bit should be two lines down, above the `Run your policy` section